src/gdswzd_mod.f90

```diff
@@ -272,236 +474,192 @@ SUBROUTINE GDSWZD_2D_ARRAY(IGDTNUM,IGDTMPL,IGDTLEN,IOPT,NPTS,FILL, &
+  !! - (15): Latitude of last grid point, octets 56-59.
+  !! - (16): Longitude of last grid point, octets 60-63.
+  !! - (17): i-direction increment, octets 64-67.
+  !! - (18): j-direction increment, octets 68-71.
+  !! - (19): Scanning mode, octet 72.
+  !!
+  !! Mercator Cyclindrical:
+  !! - (8):  Number of points along a parallel, octs 31-34.
+  !! - (9):  Number of points along a meridian, octs 35-38.
+  !! - (10): Latitude of first point, octets 39-42.
+  !! - (11): Longitude of first point, octets 43-46.
+  !! - (12): Resolution and component flags, octet 47.
+  !! - (13): Tangent latitude, octets 48-51.
+  !! - (14): Latitude of last point, octets 52-55.
+  !! - (15): Longitude of last point, octets 56-59.
+  !! - (16): Scanning mode flags, octet 60.
+  !! - (17): Orientation of grid, octets 61-64.
+  !! - (18): Longitudinal grid length, octets 65-68.
+  !! - (19): Latitudinal grid length, octets 69-72.
+  !!
+  !! Lambert Conformal Conical:
+  !! - (8):  Number of points along x-axis, octs 31-34.
+  !! - (9):  Number of points along y-axis, octs 35-38.
+  !! - (10): Latitude of first point, octets 39-42.
+  !! - (11): Longitude of first point, octets 43-46.
+  !! - (12): Resolution of component flag, octet 47.
+  !! - (13): Latitude where grid lengths specified,octets 48-51.
+  !! - (14): Longitude of meridian that is parallel to y-axis, octets 52-55.
+  !! - (15): x-direction grid length, octets 56-59.
+  !! - (16): y-direction grid length, octets 60-63.
+  !! - (17): Projection center flag, octet 64.
+  !! - (18): Scanning mode, octet 65.
+  !! - (19): First tangent latitude from pole, octets 66-69.
+  !! - (20): Second tangent latitude from pole, octets 70-73.
+  !! - (21): Latitude of south pole of projection, octets 74-77.
+  !! - (22): Longitude of south pole of projection, octets 78-81.
+  !!
+  !! Gaussian Cylindrical:
+  !! - (8):  Number of points along a parallel, octs 31-34.
+  !! - (9):  Number of points along a meridian, octs 35-38.
+  !! - (10): Basic angle of initial production domain, octets 39-42.
+  !! - (11): Subdivisions of basic angle, octets 43-46.
+  !! - (12): Latitude of first grid point, octets 47-50.
+  !! - (13): Longitude of first grid point, octets 51-54.
+  !! - (14): Resolution and component flags, octet 55.
+  !! - (15): Latitude of last grid point, octets 56-59.
+  !! - (16): Longitude of last grid point, octets 60-63.
+  !! - (17): i-direction increment, octets 64-67.
+  !! - (18): Number of parallels between pole and equator, octets 68-71.
+  !! - (19): Scanning mode, octet 72.
+  !!
+  !! Polar Stereographic Azimuthal:
+  !! - (8):  Number of points along x-axis, octets 31-34.
+  !! - (9):  Number of points along y-axis, octets 35-38.
+  !! - (10): Latitude of first grid point, octets 39-42.
+  !! - (11): Longitude of first grid point, octets 43-46.
+  !! - (12): Resolution and component flags, octet 47.
+  !! - (13): True latitude, octets 48-51.
+  !! - (14): Orientation longitude, octets 52-55.
```

Comment:
Take away the parens around the numbers, and the colon. So:
`  !! - 14 Orientation longitude, octets 52-55.
`

Reply:
Parentheses sure, but why get rid of the colon? It makes it easier to read. "Array index 11 is the longitude of first grid point, which corresponds to octets 43-46 of the grib descriptor"